core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.spec.ts

```diff
@@ -18,7 +18,9 @@
 
 import { EventEmitter } from '@angular/core';
 import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { of, Subscription } from 'rxjs';
+import { UserService } from 'services/user.service.ts';
```

Comment:
Add a newline here. [i.e, there should be a newline between external and local imports.]

Reply:
@subedi90, is this comment resolved? [In general, let the reviewer mark the comments resolved! :)]

Reply:
done